tests/unit/amo/components/TestAddonReviewListItem.js

```diff
@@ -526,6 +526,14 @@ describe(__filename, () => {
     expect(root.find('.AddonReviewListItem-begin-reply')).toHaveLength(0);
   });
 
+  it('hides the reply-to-review link on the developer reply', () => {
```

Comment:
Thanks for adding the test @rebmullin. I think this test would make more sense nested inside the `describe('Developer reply to a review')`.

Reply:
thanks @bobsilverberg  - good pt